### ISSUE TITLE
Intellisense for latest release (0.12.3) not working with VS code

### ISSUE THREAD

**Issue by snallamoansys** [pyansys/pyfluent#1333]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

I am not seeing the intellisense working in visual studio code. Anything else needs to be done? For previous release (0.11.0), it works fine as shown here. But for 0.12.3, nothing appears after solver.setup.

![image](https://user-images.githubusercontent.com/99287292/217265891-b77d5ecb-6ecd-4b48-8b5d-844cda0de477.png)

![image](https://user-images.githubusercontent.com/99287292/217265670-9d182017-1f71-4438-9203-594ebd0cee00.png)

### 📝 Steps to reproduce

In Visual Studio Code, activate virtual environment with 0.12.3 version of ansys-fluent-core installed and load any case file such as elbow case file and try to access the settings api as shown in the picture

import ansys.fluent.core as pyfluent
fluent = pyfluent.launch_fluent(mode='solver',show_gui=True)
tui=fluent.tui
fluent.file.read(file_name=r'elbow.cas.h5',file_type='case')

### 💻 Which operating system are you using?

Windows

### 🐍 Which Python version are you using?

3.9

### 📦 Installed packages

```shell
ansys-api-fluent==0.3.5
ansys-api-platform-instancemanagement==1.0.0b3
ansys-fluent-core==0.12.3
ansys-fluent-parametric==0.6.0
ansys-fluent-visualization==0.6.0
ansys-platform-instancemanagement==1.0.2
appdirs==1.4.4
asttokens==2.2.1
backcall==0.2.0
certifi==2022.12.7
charset-normalizer==3.0.1
click==8.1.3
colorama==0.4.6
comm==0.1.2
contourpy==1.0.7
cycler==0.11.0
dash==2.7.1
dash-bootstrap-components==1.3.0
dash-core-components==2.0.0
dash-html-components==2.0.0
dash-table==5.0.0
debugpy==1.6.5
decorator==5.1.1
entrypoints==0.4
executing==1.2.0
Flask==2.2.2
fonttools==4.38.0
googleapis-common-protos==1.58.0
grpcio==1.51.1
h5py==3.7.0
idna==3.4
imageio==2.24.0
importlib-metadata==6.0.0
ipykernel==6.20.2
ipympl==0.9.2
ipython==8.8.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
itsdangerous==2.1.2
jedi==0.18.2
Jinja2==3.1.2
jupyter_client==7.4.9
jupyter_core==5.1.3
jupyterlab-widgets==3.0.5
kiwisolver==1.4.4
MarkupSafe==2.1.2
matplotlib==3.6.3
matplotlib-inline==0.1.6
nest-asyncio==1.5.6
numpy==1.24.1
packaging==23.0
pandas==1.5.2
parso==0.8.3
pickleshare==0.7.5
Pillow==9.4.0
platformdirs==2.6.2
plotly==5.12.0
pooch==1.6.0
prompt-toolkit==3.0.36
protobuf==3.20.3
protoc-gen-swagger==0.1.0
psutil==5.9.4
pure-eval==0.2.2
Pygments==2.14.0
pyparsing==3.0.9
PySide6==6.4.2
PySide6-Addons==6.4.2
PySide6-Essentials==6.4.2
python-dateutil==2.8.2
pytz==2022.7.1
pyvista==0.37.0
pyvistaqt==0.9.0
pywin32==305
pyzmq==25.0.0
QtPy==2.3.0
requests==2.28.2
scooby==0.7.0
shiboken6==6.4.2
six==1.16.0
stack-data==0.6.2
tenacity==8.1.0
tornado==6.2
traitlets==5.8.1
urllib3==1.26.14
vtk==9.2.5
wcwidth==0.2.6
Werkzeug==2.2.2
widgetsnbextension==4.0.5
zipp==3.11.0
```

**Comment by mkundu1** [comment]
Intellisense is working in Jupyter, so the runtime queries are ok. The static queries seem to be not working - need to check the type-hints.

**Comment by dnwillia-work** [comment]
Yeah, I can confirm this.  The settings API is not showing up.

**Comment by dnwillia-work** [comment]
FWIW, if I install pyreadline3 I can see the children.

**Comment by mkundu1** [comment]
This is broken because we now have multiple versions of settings module and dynamically choose one based on the connected Fluent version. One way to fix this is to have the 23.1 settings module chosen by default (statically) and change it at runtime if a different version of Fluent is connected.

**Comment by sravanansys** [comment]
@mkundu1 intellisense is not working for me in VS code with latest release 0.17.0. Anything else to be done after installing the release packages?

![image](https://github.com/ansys/pyfluent/assets/99287292/97dbc750-a797-4b06-9f00-c3790909224b)
